include/RED4ext/Dump/Reflection-inl.hpp

```diff
@@ -41,10 +41,20 @@ RED4EXT_INLINE void Dump(std::filesystem::path aOutPath, std::filesystem::path a
     {
         size_t i = 0;
 
-        // Special case for AI
-        if (aInput.size() >= 2 && aInput[0] == 'A' && aInput[1] == 'I')
+        constexpr std::string_view exceptions[] = {"AI", "GpuWrapApi", "GpuWrapApiVertexPacking"};
```

Comment:
```suggestion
        static constexpr std::string_view specialNamespaces[] = {"AI", "GpuWrapApi", "GpuWrapApiVertexPacking"};
```

Or something better than `specialNamespaces`.

Reply:
`GpuWrapApiVertexPacking` contains the `GpuWrapApi` substring. Is it necessary to have it?

Reply:
For the sake of not having to do extra logic to handle `VertexPacking` as a subnamespace of `GpuWrapApi`, yes it's necessary